db/migrate/20210602115738_add_opt_out_of_updates_to_school_cohorts.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddOptOutOfUpdatesToSchoolCohorts < ActiveRecord::Migration[6.1]
+  def change
+    add_column :school_cohorts, :opt_out_of_updates, :boolean, null: false, default: false
```

Comment:
Is this so we can have people opt out for other reasons? At the moment we can work this out based on the value of `induction_programme_choice`, right? (Just checking my understanding)

Reply:
Yes, thought it made sense to separate the programme choice to the opt out